content/en/docs/Tasks/creating-operator-manifests.md

```diff
@@ -268,15 +268,14 @@ However, the following is a suggested guideline to follow when including Priorit
 #### Extension apiservers and APIServices
 
 Please see the document on [extension apiservers]() if your operator does not rely on CRDs to provide its API.
+#### Advanced and Optional features
 
-#### Operator SDK
+Please see the documentation for [advanced operator configuration]() which includes additional suggestions for further integration with OLM.
```

Comment:
Missing link?

Reply:
Doc isn't ready yet :( Placeholder for future doc

Reply:
Hmm, wdyt about putting an explicit TODO here (and in other places withe empty links).
Having this implicit might be confusing to a user who would think the link is just broken.

And being explicit also makes it a more obvious reminder for us to address this later on.

Reply:
That's a good idea. Replacing these with TODOs.